maas/client/facade.py

```diff
@@ -139,6 +139,7 @@ def machines(origin):
             "allocate": origin.Machines.allocate,
             "get": origin.Machine.read,
             "list": origin.Machines.read,
+            "power_parameters": origin.Machines.power_parameters,
```

Comment:
`allocate`, `get`, and `list` are all verbs/actions here, so this should be `get_power_parameters` or something similar.